content/en/docs/demo/collector-data-flow-dashboard.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Collector Data Flow Dashboard
+---
+
+Monitoring data flow through the OpenTelemetry Collector is crucial for several
+reasons. Gaining a macro-level perspective on incoming data, such as sample
+counts and cardinality, is essential for comprehending the collector's internal
+dynamics. However, when delving into the details, the interconnections can
+become complex. The Collector Data Flow Dashboard aims to demonstrate the
+capabilities of the OpenTelemetry demo application, offering a solid foundation
+for users to build upon.
+
+## Overview
+
+![](/docs/demo/screenshots/otelcol-data-flow-overview.png)
```

Comment:
All images (here and elsewhere in the document) should have (nonempty) `alt` text (inside the square brackets).

Reply:
Marking as unresolved since I'm not seeing any alt text.

Reply:
Move the image after the opening paragraph.